### DISCUSSION TITLE
Advance intent for closing the app?

### DISCUSSION THREAD

**Discussion by Recress** [Nain57/Smart-AutoClicker#268] [Q&A]
Hello!

I would like to know how to properly utilize the advanced function of this amazing app? The intention is the app is sometimes unresponding, so my goal is to close it and open the app again.

**Comment by Recress** [comment]
I was hoping someone would help me. Was thinking to restart the app every 30 mins or so..

**Comment by Nain57** [comment]
Hi,

As far as I know, there is no options in the [Android API for Intents](https://developer.android.com/reference/android/content/Intent) that allows to kill/restart an application. The whole API should be  implemented in Smart AutoClicker, but I might be wrong, if you find something, let me know.

**Reply by Recress** [reply]
Hello @Nain57, yes what I did is download an app that closes the application instead and put an image scanning.

**Reply by Recress** [reply]
Oh I forgot to mention, seems like the Autoclicker doesnt work on Bluestacks..? @Nain57
